rhobotSetup.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 set -x #echo on
 
-cp $GOLANGDEPDIR/rhobotGolangDependencies.tar.gz $WORKSPACE/rhobotGolangDependencies.tar.gz
-wget --quiet -O $WORKSPACE/go1.6.2.linux-amd64.tar.gz https://storage.googleapis.com/golang/go1.6.2.linux-amd64.tar.gz
+wget --quiet -O $WORKSPACE/go1.8.3.linux-amd64.tar.gz https://storage.googleapis.com/golang/go1.8.3.linux-amd64.tar.gz
+mkdir $WORKSPACE/rhobot-for-el6/go -p
+tar -xzf $WORKSPACE/go1.8.3.linux-amd64.tar.gz -C $WORKSPACE/rhobot/
 
-mkdir $WORKSPACE/rhobot -p
-tar -xzf $WORKSPACE/rhobotGolangDependencies.tar.gz -C $WORKSPACE/rhobot
-tar -xzf $WORKSPACE/go1.6.2.linux-amd64.tar.gz -C $WORKSPACE/rhobot/
+# mkdir $WORKSPACE/rhobot-for-el6/gopath/src/github.com/cfpb/rhobot -p
+# cp -rpf $WORKSPACE/rhobot $WORKSPACE/rhobot-for-el6/gopath/src/github.com/cfpb/rhobot
```

Comment:
@frank0631 Can we just get rid of these lines, or are they still useful?  If they are useful, could you provide context in a comment?